src/openeo_test_suite/tests/workflows/test_add_dimension.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+
+import xarray as xr
+
+
+def test_ndvi_add_dim(cube_one_day_red_nir, b_dim, tmp_path):
+    filename = tmp_path / "test_ndvi_add_dim.nc"
+
+    def compute_ndvi(data):
+        from openeo.processes import array_element
+
+        red = data.array_element(index=0)
+        nir = data.array_element(index=1)
```

Comment:
FYI: you don't necessarily need array_element (or import it from openeo.processes), normally this should work too:

```
red = data[0]
nir = data[1]
```